src/main/java/com/mixpanel/android/mpmetrics/MixpanelPushNotification.java

```diff
@@ -188,8 +189,8 @@ protected void buildNotificationFromData() {
         final PendingIntent contentIntent = PendingIntent.getActivity(
                 context,
                 0,
-                data.intent,
-                PendingIntent.FLAG_UPDATE_CURRENT
+                getRoutingIntent(data.onTap, "notificationClick", "notification"),
```

Comment:
as mentioned in another PR, I think we should just be passing the `target` (as an enum) to `getRoutingIntent` and when the target is `notification`, we don't need an id or label.